modules/internal/String.chpl

```diff
@@ -1911,6 +1959,21 @@ module String {
     return s;
   }
 
+  /*
+     :returns: A string with complete multibyte character
+               with the codepoint value `i`.
```

Comment:
Maybe "A string storing the Unicode code point `i`". Also consider renaming the argument to `codepoint`.